seatunnel-connectors-v2/connector-mongodb/src/main/java/org/apache/seatunnel/connectors/seatunnel/mongodb/config/MongodbConfig.java

```diff
@@ -104,6 +104,12 @@ public class MongodbConfig {
                     .withDescription(
                             "By utilizing flatSyncString, only one field attribute value can be set, and the field type must be a String. This operation will perform a string mapping on a single MongoDB data entry.");
 
+    public static final Option<Boolean> ALLOW_NULL =
+            Options.key("allow-null")
+                    .booleanType()
+                    .defaultValue(false)
+                    .withDescription("allow null value when fields reading is null");
+
     // --------------------------------------------------------------------
```

Comment:
Please describe this parameter in the documentation.
https://github.com/apache/seatunnel/blob/dev/docs/en/connector-v2/source/MongoDB.md